docs/reference/apilinks.rst

```diff
@@ -0,0 +1,16 @@
+API self-referencing links
+==========================
+
+Some API endpoints can provide additional information or context by returning links to other API endpoints.
+The following table documents all these links and their format:
+
+
+.. list-table:: Self-referencing links
```

Comment:
Can we reference this from the api spec itself, where it is used?

Reply:
Added a reference to the description of the endpoints (see image below). I tried adding this reference further down the tree, e.g. in the description of the uri of the response object, but I didn't managed to.
![image](https://github.com/inmanta/inmanta-core/assets/98876549/bba0c32a-de12-4766-94fd-4aba2f5ac2cf)

Reply:
Good enough I guess.